src/commonMain/kotlin/au/id/micolous/metrodroid/card/desfire/DesfireCardTransitRegistry.kt

```diff
@@ -33,6 +33,7 @@ object DesfireCardTransitRegistry {
             MagnaCartaTransitData.FACTORY,
             IntercardTransitData.FACTORY,
             TPFCardTransitData.Factory,
-            BlankDesfireTransitFactory,
-            UnauthorizedDesfireTransitData.FACTORY)
+            BlankDesfireTransitData.FACTORY,
+            UnauthorizedDesfireTransitData.FACTORY,
+            HoloTransitData.FACTORY)
```

Comment:
`HoloTransitData.FACTORY` should be directly after `TPFCardTransitData`. `BlankDesfireTransitData` and `UnauthorizedDesfireTransitData` should be the last two entries as they are a last resort.